### PR TITLE
Replace BB verbiage with BFD verbiage.

### PR DESCRIPTION
**Pull request by jmcconn120**
Replacing Blue Button verbiage with BFD

### Proposed Changes

Just replaced any relevant mentions of BB with BFD language instead. 
### Change Details

Simple find and replace, N/A
### Security Implications

N/A
- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### Feedback Requested

Just check to make sure that the language for BFD makes sense.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -9,14 +9,13 @@ subnav-link-gradient: "blueberry-lime-link"
 nav_link: home
 id: home
 sections:
-  - Overview
```
Comment (msnook) [inline index.md]: Why was this removed?

File: index.md [hunk 2]
```diff
@@ -9,14 +9,13 @@ subnav-link-gradient: "blueberry-lime-link"
...
   - Getting Started
   - Frequently Asked Questions
   - FHIR APIs at CMS
 ---
 
 
-## Overview
+# Overview
```
Comment (msnook) [inline index.md]: Why was this edited?

File: index.md [hunk 3]
```diff
@@ -68,7 +67,7 @@ Representatives of Shared Savings Program ACOs are now invited to express their
...
-    * **FHIR:** FHIR (Fast Healthcare Interoperability Resources) is a specification for exchanging healthcare data electronically, allowing any system to access and consume this data to solve clinical and administrative problems around healthcare-related data. BCDA is built on the Blue Button 2.0 API, which has structured its data using the FHIR standard. BCDA will use three resource types from the FHIR specification: explanation of benefits, patient, and coverage. Learn more about FHIR in the [guide to working with BCDA data.](/data-guide/){:target="_self"}
+    * **FHIR:** FHIR (Fast Healthcare Interoperability Resources) is a specification for exchanging healthcare data electronically, allowing any system to access and consume this data to solve clinical and administrative problems around healthcare-related data. BCDA is built on the Beneficiary Fhir Data Server (BFD) API, which has structured its data using the FHIR standard. BCDA will use three resource types from the FHIR specification: explanation of benefits, patient, and coverage. Learn more about FHIR in the [guide to working with BCDA data.](/data-guide/){:target="_self"}
```
Comment (msnook) [inline index.md]: Should "Fhir" be "FHIR"?

File: index.md (hunk 1 above)
Comment (jmcconn120) [reply index.md]: Umm It shouldn't have been? I did a simple find and then replaced, so not sure how that happened. Will change.

Comment (msnook) [reply index.md]: This section is still removed in latest push.  Need to make sure this gets added back in.
